modules/ROOT/pages/redelivery-policy.adoc

```diff
@@ -0,0 +1,62 @@
+= Redelivery Policy
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint studio, flow, redelivery, policy
+
+A redelivery policy is a filter that helps you conserve resources by limiting the number of times the Mule runtime engine (Mule) executes flow messages that cause errors. You can add a redelivery policy to some connectors that act as flow sources, such as the Salesforce, SAP, and VM connectors. To see whether a connector can use a redelivery policy, see the reference document for that connector.
+
+When you add a redelivery policy to a flow's source, Mule evaluates the received data before it executes the flow's components. If the data caused failures a specified number of times, the redelivery policy prevents the flow from processing the received data and raises a REDELIVERY_EXHAUSTED error.
+
+Some connectors have their own logic for handling an error thrown by a redelivery policy. In such cases, check the documentation for the connector to which the source belongs. For example, the HTTP connector might return 429: TOO MANY REQUESTS.
```

Comment:
PRovide a link so people can see what doc explains this.

Reply:
I took out the example. Users will have to check the reference doc for their connector,